sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/google-oauth2.yaml

```
@@ -225,10 +261,17 @@ provider:
       detection:
         matching:
           # "new google.auth.OAuth2("
-          # first parameter (client ID) is a quoted string ending in ".apps.googleusercontent.com"
-          # optional "//" comment after the client ID
-          # second parameter (client secret) is a quoted string
-          pattern: "new\\s+google\\.auth\\.OAuth2\\(\\s*[\"'][a-z0-9\\-]+\\.apps\\.googleusercontent\\.com[\"']\\s*,\\s*+(?://[^\r\n]+\\s*+)?[\"']([A-Za-z0-9_\\-]{24})[\"']"
+          # - first parameter (client ID) is a quoted string ending in ".apps.googleusercontent.com"
+          # - optional "//" comment after the client ID
+          # - second parameter (client secret) is a quoted string
+          pattern: "new\\s++google\\.auth\\.OAuth2\\(\\s*+\
+                    [\"']\
+                        [a-z0-9\\-]++\\.apps\\.googleusercontent\\.com\
+                    [\"']\
+                    \\s*+,\\s*+(?://[^\r\n]++\\s*+)?\
```

It seems to work, but we usually "double escape" \n and \r.
So I suggest either adding the additional backslash here for consistency or removing the additional backslash from the other instances.
```suggestion
                    \\s*+,\\s*+(?://[^\\r\\n]++\\s*+)?\
```